packages/core/addon/models/delivery-rule.js

```diff
@@ -37,6 +37,7 @@ export default DS.Model.extend(Validations, {
   format: DS.attr({ defaultValue: () => {} }),
   recipients: DS.attr({ defaultValue: () => [] }),
   version: DS.attr('number', { defaultValue: '1' }),
+  notifyWhenEmpty: DS.attr('boolean', { defaultValue: false }),
```

Comment:
Can we move this to the `schedulingRules` and call it `mustHaveData` with a default of `false`

`schedulingRules` is intended to be where we put advanced settings.